### PR TITLE
Add notify and color options to HipchatService

### PR DESCRIPTION
**Pull request by dsander**
When notify is set to true send messages will trigger a notification for all room members.
Color changes the background color of the message.

### CODE REVIEW

File: app/models/project_services/hipchat_service.rb [hunk 1]
```diff
@@ -39,6 +39,8 @@ def fields
     [
       { type: 'text', name: 'token',     placeholder: 'Room token' },
       { type: 'text', name: 'room',      placeholder: 'Room name or ID' },
+      { type: 'checkbox', name: 'notify' },
+      { type: 'select', name: 'color', choices: ['yellow', 'red', 'green', 'purple', 'gray', 'random'] },
```
Comment (jvanbaarsen) [inline app/models/project_services/hipchat_service.rb]: Are these colors picked by you? Or are they provided by Hipchat?

Comment (dsander) [inline app/models/project_services/hipchat_service.rb]: Those are the colors that are suppored by [HipChat](https://www.hipchat.com/docs/apiv2/method/send_room_notification).

**Comment by jvanbaarsen** [comment]
@dsander Thanks! Can you add an entry to the changelog? Ping me when that is done so I can merge this.

File: app/models/project_services/hipchat_service.rb (hunk 1 above)
Comment (jvanbaarsen) [inline app/models/project_services/hipchat_service.rb]: @dsander Ok!

**Comment by dsander** [comment]
@jvanbaarsen Done, sorry totally forgot about that.

**Comment by jvanbaarsen** [comment]
Thanks!

**Comment by dsander** [comment]
Thanks for merging :smile:

**Pull request merged**